README.md

```diff
@@ -24,7 +24,119 @@ $ helm install nfs-subdir-external-provisioner nfs-subdir-external-provisioner/n
     --set nfs.path=/exported/path
 ```
 
-### Without Helm
+### With Kustomize
+
+**Step 1: Get connection information for your NFS server**
+
+Make sure your NFS server is accessible from your Kubernetes cluster and get the information you need to connect to it. At a minimum you will need its hostname.   
+
+**Step 2: Add the base resource**
+
+Create a `kustomization.yaml` file in a directory of your choice, and add the [deploy](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/tree/master/deploy) directory as a resource. This will use the kustomization file within that folder as our base.
+
+```yaml
+namespace: nfs-provisioner
+resources:
+  - github.com/kubernetes-sigs/nfs-subdir-external-provisioner//deploy
```

Comment:
double `//`

Reply:
Required since Kustomize uses Hashicorp's go-getter URL format. `//` is used to specify a repository folder.